pkgs/development/python-modules/pypblib/default.nix

```diff
@@ -0,0 +1,18 @@
+{ lib, buildPythonPackage, fetchPypi }:
+
+buildPythonPackage rec {
+	pname = "pypblib";
+	version = "0.0.4";
+
+	src = fetchPypi {
+		inherit pname version;
+		sha256 = "cd2TC/F3yjjW7rRzcC0F3wfhHyA4LbDHZkZSl+r0kGI=";
```

Comment:
There's still a preference to having base32 hashes within nixpkgs to ease the upgrade process for users with holder versions of the nix cli.

You can use `nix-hash --to-base32 <hash>` to convert the hash into the related base32 hash.

For more details refer to: https://github.com/NixOS/nixpkgs/pull/89308#issuecomment-636927054 #89275